content/en/blog/2025/kubecon-eu.md

```diff
@@ -0,0 +1,220 @@
+  by Harry Kimpel, New Relic<br> Tuesday, April 1 • 10:00 - 10:05
+- **[Sponsored Keynote: Fluent Bit vs. OpenTelemetry Collector - Why Not Both?](https://sched.co/1u5jp)**<br>
+  by Paige Cruz, Chronosphere<br> Tuesday, April 1 • 10:10 - 10:15
+- **[Observability by Design: Leveraging OpenTelemetry Weaver To Take Control of Your Observability](https://sched.co/1u5jy)**<br>
+  by Josh Suereth, Google & Laurent Querel, F5<br> Tuesday, April 1 • 11:20 -
+  11:45
+- **[Encrypted Telemetry in Transit](https://sched.co/1u5k4)**<br> by Jason
+  Plumb, Splunk<br> Tuesday, April 1 • 12:55 - 13:20
+- **[From Sampling To Full Visibility: Scaling Tracing To Trillions of Spans](https://sched.co/1u5k7)**<br>
+  by Sonam Gupta, SigLens<br> Tuesday, April 1 • 12:55 - 13:20
+- **[OpenTelemetry at Delivery Hero: The Good, the Bad and the Vendor-Agnostic](https://sched.co/1u5kD)**<br>
+  by Elena Kovalenko, Delivery Hero<br> Tuesday, April 1 • 13:30 - 13:55
+- **[From Splunk To OTel: Scaling Observability at MSCI With a Four-Person Team](https://sched.co/1u5kP)**<br>
+  by Aftab Khan & Zach Arnold, MSCI<br> Tuesday, April 1 • 14:40 - 15:05
+- **[Customize Your Own OpenTelemetry Collector: An Introduction To OCB](https://sched.co/1u5kV)**<br>
+  by Evan Bradley, Dynatrace & Pablo Baeyens, Datadog<br> Tuesday, April 1 •
+  15:20 - 15:45
+- **[The Art and Craft of No-touch Instrumentation](https://sched.co/1u5kS)**<br>
+  by Michele Mancioppi, Dash0<br> Tuesday, April 1 • 15:20 - 15:45
+- **[OTel-y Oops: Learning From Our Observability Blunders](https://sched.co/1u5kY)**<br>
+  by Joe Stephenson & Rodney Karemba, Akamai Technologies<br> Tuesday, April 1 •
+  15:55 - 16:20
+- **[Beyond the Ephemeral: Mastering Serverless Metrics at Scale With Shopify](https://sched.co/1u5kh)**<br>
+  by Pedro Tanaka, Shopify<br> Tuesday, April 1 • 16:30 - 16:55
+- **[The State of OpenTelemetry Profiling](https://sched.co/1u5ke)**<br> by
+  Damien Matthieu, Elastic & Felix Geisendoerfer, Datadog<br> Tuesday, April 1 •
+  16:30 - 16:55
+- **[⚡ Lightning Talk: Unlocking Customer-Centric Observability: A Case Study of OpenTelemetry To Reduce MTTD < 3 Mins](https://sched.co/1u5kn)**<br>
+  by Kokilavani Kathiresan, Intuit<br> Tuesday, April 1 • 17:00 - 17:10
+- **[⚡ Lightning Talk: Who Are You? Solving the Container ID Resolution With Cgroupv2](https://sched.co/1u5kk)**<br>
+  by Vincent Boulineau, Datadog<br> Tuesday, April 1 • 17:00 - 17:10
+- **[⚡ Lightning Talk: Empowering OpenTelemetry Users With the OTTL Playground: Simplified Data Transformation and Testing](https://sched.co/1u5kt)**<br>
+  by Edmo Vamerlatti Costa, Elastic<br> Tuesday, April 1 • 17:15 - 17:25
+- **[⚡ Lightning Talk: From HAR To OpenTelemetry Trace: Redefining Your Observability](https://sched.co/1u5kq)**<br>
+  by Antonio Jimenez Martinez, Cisco ThousandEyes<br> Tuesday, April 1 • 17:15 -
+  17:25
+
+<!-- ## OpenTelemetry Observatory
```

Comment:
@open-telemetry/sig-end-user-approvers, does anyone know if the Observatory is happening again this year? I couldn't find any mention of it on the Kubecon or Observability Day sites. Thanks!